src/daml/Daml/Finance/App/Lending/Model.daml

```diff
@@ -0,0 +1,76 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.App.Lending.Model where
+
+import DA.Set (singleton)
+import Daml.Finance.Interface.Holding.Account qualified as Account (K, exerciseInterfaceByKey)
+import Daml.Finance.Interface.Holding.Lockable qualified as Lockable (I, Release(..))
+import Daml.Finance.Interface.Holding.Transferable qualified as Transferable (I, Transfer(..))
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (Q)
+import Daml.Finance.Interface.Util.Disclosure qualified as Disclosure (I, RemoveObservers(..))
+
+template BorrowOfferRequest
+  with
+    operator : Party
+    provider : Party
+    customer : Party
+    id : Text
```

Comment:
Use `Id` from common

Reply:
Yeah, I'll do that in a separate PR. The whole codebase uses `Text` currently.

Reply:
👍